src/components/Recipes/RecipePage.tsx

```diff
@@ -0,0 +1,35 @@
+import { useParams } from "react-router-dom";
+import React, { useEffect, useState } from "react";
+import "./RecipePage.css"
+import { readRecipe } from "../../utils/SpoonacularRequests"
+import RecipeInfo from "../../models/recipeInfo";
+
+const RecipePage: React.FC = () => {
+    const {recipeID} = useParams();
+    const [recipeData, setRecipeData] = useState<RecipeInfo | null>(null);
+
+    useEffect(() => {
+        if (recipeID) {
+        readRecipe(recipeID).then((response) => setRecipeData(response))
+        }
+    }, [recipeID]);
+
+    
+    if (recipeData) {
+    return (
+        <main>
+            <figure>
+        <img src={recipeData.image} alt={recipeData.title}/>
+            </figure>
+        <h1>{recipeData.title}</h1>
+        <p>{recipeData.instructions}</p>
+        </main>
```

Comment:
our linter is a little messed up Lol -- but please put this in a html tree format  w/ proper indents